units/systemd-udev-settle.service.in

```diff
@@ -24,4 +24,5 @@ ConditionPathIsReadWrite=/sys
 Type=oneshot
 TimeoutSec=180
 RemainAfterExit=yes
+Environment=SYSTEMD_SERVICE=%n
```

Comment:
I think the sd_pid_get_unit() approach mentioned above is nicer. That said we really should look into passing a SYSTEMD_UNIT= env var to all processes we fork off possibly, but probably in a different PR.